src/api/IBMi.ts

```diff
@@ -784,7 +797,7 @@ export default class IBMi {
                                   if (pathRegex) {
                                     bashrcContent[index] = `${pathRegex[1]}/QOpenSys/pkgs/bin:${pathRegex[2]
                                       .replace("/QOpenSys/pkgs/bin", "") //Removes /QOpenSys/pkgs/bin wherever it is
-                                      .replace("::", ":")}`; //Removes double : in case /QOpenSys/pkgs/bin wasn't at the end
+                                      .replace("::", ":")}/QOpenSys/usr/bin:/usr/bin`; //Removes double : in case /QOpenSys/pkgs/bin wasn't at the end
```

Comment:
@jonnyz32 You're missing a `:` here:
```suggestion
                                      .replace("::", ":")}:/QOpenSys/usr/bin:/usr/bin`; //Removes double : in case /QOpenSys/pkgs/bin wasn't at the end
```

I tested with a PATH not having `/QOpenSys/usr/bin` in it, and I ended up with this:
```shell
export PATH=/QOpenSys/pkgs/bin:/usr/bin/QOpenSys/usr/bin:/usr/bin
```
Note the `/usr/bin/QOpenSys/usr/bin` in the path!